docs/book/src/developers/index.md

```diff
@@ -0,0 +1,5 @@
+# Developer Guide
+
+Everything you need to know about contributing to CAPRKE2.
+
+If you are new to the project and want to help but don’t know where to start, you can refer to the [Cluster API contributing guide](https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md).
```

Comment:
we can't refer to CAPI contributing guide but rather have our own first and then link to it. For now it can be omitted